terraform/aks/application.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  environment  = "${var.environment}${var.pr_number}"
+}
+
+module "application_configuration" {
+  source = "./vendor/modules/aks//aks/application_configuration"
+
+  namespace              = var.namespace
+  environment            = local.environment
+  azure_resource_prefix  = var.azure_resource_prefix
+  service_short          = var.service_short
+  config_short           = var.config_short
+  secret_key_vault_short = "app"
+
+  # Delete for non rails apps
+  is_rails_application = true
+
+  config_variables = {
+    PGSSLMODE        = local.postgres_ssl_mode
+  }
+  secret_variables = {
+    DATABASE_URL = module.postgres.url
+    REDIS_URL    = module.redis-cache.url
+# below added from paas config
+    HTTPAUTH_PASSWORD = module.infrastructure_secrets.map.HTTP-PASSWORD,
+    HTTPAUTH_USERNAME = module.infrastructure_secrets.map.HTTP-USERNAME,
+    BASIC_AUTH        = var.basic_auth,
+    APP_URL           = length(var.paas_asset_hostnames) == 0 ? "" : "https://${var.paas_internet_hostnames[0]}.education.gov.uk",
+#   keeping here as a reminder, but went be set in aks and need to confirm impact
+    APP_ASSETS_URL    = length(var.paas_asset_hostnames) == 0 ? "" : "https://${var.paas_asset_hostnames[0]}.education.gov.uk"
+  }
+}
+
+module "web_application" {
+  source = "./vendor/modules/aks//aks/application"
+
+  is_web = true
+
+  namespace    = var.namespace
+  environment  = local.environment
+  service_name = var.service_name
+  probe_path   = "/healthcheck.json"
```

Comment:
The healthcheck.json may be too heavy for the probe as it checks all the dependencies, as per: https://github.com/DFE-Digital/get-into-teaching-app/blob/4aa365ce19dd09d6da9c08ba14e52ec9d4abac06/app/models/healthcheck.rb#L4
Can we replace it with a simple one like: https://github.com/DFE-Digital/teacher-pay-calculator/blob/main/config/routes.rb#L11